api.go

```diff
@@ -492,9 +492,26 @@ func NewRaft(conf *Config, fsm FSM, logs LogStore, stable StableStore, snaps Sna
 		}
 		r.processConfigurationLogEntry(&entry)
 	}
+
 	r.logger.Printf("[INFO] raft: Initial configuration (index=%d): %+v",
 		r.configurations.latestIndex, r.configurations.latest.Servers)
 
+	mismatchedIP := false
```

Comment:
I don't think we want to do this since each server will only fix themselves, and it will result in each of them having a different configuration, which seems dangerous.